src/test/java/pages/HomePage.java

```diff
@@ -14,9 +16,51 @@ public HomePage(WebDriver givenDriver){
 
     //Locator
     By userAvatarIcon = By.cssSelector("img.avatar");
+    By addPlayList = By.cssSelector("i[data-testid='sidebar-create-playlist-btn']");
+    By createPlayListBtn = By.cssSelector("li[data-testid='playlist-context-menu-create-simple']");
+    By playListNameInput = By.cssSelector("[name='create-simple-playlist-form']>input");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```